libs/blocks/iframe/iframe.js

```diff
@@ -0,0 +1,13 @@
+import { createTag } from '../../utils/utils.js';
+
+export default function init(link) {
+  const url = link.href ?? link.querySelector('a')?.href;
```

Comment:
You could remove the optional chaining here I think as well. If the link does not have an href, it will return `undefined` which will lead ` if(!url)`  to evaluate to true, thus fulfilling your condition regardless of the chaining.

Reply:
Optional chaining is required here, if a link is not authored then you will get an type error because it will resolve to `null.href`.

Reply:
This seems really weird here. Where is the scenario where a link is inside another link?

@Brandon32

Reply:
To test this functionality I am creating a block with a link in it, so `link` is a `div > a`. Normally init takes the variable `a` for autoBlocks or `el` for blocks, I used `link`, maybe `el` is a better option?

Reply:
@auniverseaway I've updated the code to use `el`, let me know if that work for you, thanks.